src/components/calculator.js

```diff
@@ -0,0 +1,26 @@
+const Calculator = () => (
+  <div className="cal-container">
+    <p className="cal-header">0</p>
+    <p className="keys">AC</p>
+    <p className="keys">+/-</p>
+    <p className="keys">%</p>
+    <p className="keys right-col">÷</p>
+    <p className="keys">7</p>
+    <p className="keys">8</p>
+    <p className="keys">9</p>
+    <p className="keys right-col">x</p>
+    <p className="keys">4</p>
+    <p className="keys">5</p>
+    <p className="keys">6</p>
+    <p className="keys right-col">-</p>
+    <p className="keys">1</p>
+    <p className="keys">2</p>
+    <p className="keys">3</p>
+    <p className="keys right-col">+</p>
+    <p className="bottom-zero">0</p>
+    <p className="keys">.</p>
+    <p className="keys right-col">=</p>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- [ ] The Calculator should have at least one child component. Kindly implement that.

  Perhaps you can make these buttons into one reusable component and import it here.

Reply:
Hello @julie-ify, I think I'm not supposed to delete reportwebvitals.js and setuptests.js files. As I delete them my program doesn't run and even this is not mention in requirements.